src/plugins/intel_gpu/src/graph/include/serialization/object_types.hpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+namespace cldnn {
+
+enum class object_type {
+    ACTIVATION_IMPL,
```

Comment:
Can we get rid of yet another list of primitives somehow?

Reply:
Removed. Thank you.